tests/album-create.test.js

```diff
@@ -0,0 +1,34 @@
+const { expect } = require('chai')
```

Comment:
In all of your test files there are keywords highlighted in yellow. This doesn't actually cause any functional issues but if you want to get rid of it then you need to add `"mocha": true` to the eslintrc.js file, underneath `"es2021": true,` would be fine.